package.json

```diff
@@ -32,7 +32,7 @@
     "Doron Tsur <doront@wix.com> (http://github.com/qballer)",
     "Gady Guy <gadig@wix.com> (https://github.com/gadiguy)"
   ],
-  "license": "SEE LICENSE IN LICENSE.md",
+  "license": "BSD",
```

Comment:
taking a second look here: https://docs.npmjs.com/files/package.json#license

you are right its fine to keep it the way it was.


On Mon, Mar 21, 2016 at 11:03 AM, Doron Tsur <notifications@github.com>
wrote:

> In package.json
> <https://github.com/wix/tspoon/pull/59#discussion_r56794048>:
>
> > @@ -32,7 +32,7 @@
> >      "Doron Tsur <doront@wix.com> (http://github.com/qballer)",
> >      "Gady Guy <gadig@wix.com> (https://github.com/gadiguy)"
> >    ],
> > -  "license": "SEE LICENSE IN LICENSE.md",
> > +  "license": "BSD",
>
> You have to refer to the license file because it is a special BSD version.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/wix/tspoon/pull/59/files/1dffca2c6b78124e1ad83e414dfc9793d3fbaf01#r56794048>
>